common/api/appui-react.api.md

```diff
@@ -4665,7 +4667,7 @@ export enum ToolbarUsage {
     ViewNavigation = 1
 }
 
-// @beta
+// @beta @deprecated
```

Comment:
Need a codemod to change `ToolbarWithOverflow` to `Toolbar` with `enableOverflow` prop.

Reply:
Are we planning to codemod every changes now ?

Reply:
I assume ideally we'd want to deliver a codemod for each deprecation.
We could then mention in the NextVersion.md to run the codemod to facilitate the upgrade :)
ATM we only have one version v4.0.0 so just go ahead and create a separate `v4.4.0` dir. 
Additionally, I was thinking it would make sense for 5.0 codemod to call all 4.x codemods :)

Reply:
Created #446 to enable this support, learning this with a basic level of confidence for this PR will delay a bit too much.